pyproject.toml

```diff
@@ -58,6 +58,7 @@ smart-open = {version = ">=6.3.0", extras = ["s3"], optional = true}
 jina-hubble-sdk = {version = ">=0.34.0", optional = true}
 elastic-transport = {version ="^8.4.0", optional = true }
 qdrant-client = {version = ">=1.1.4", python = "<3.12", optional = true }
+jax = {version = ">=0.4.10", optional = true}
```

Comment:
can you add as well an `extra` section that install jax ?

it is right below this line.

And you need to add it to the all section as well